src/js/video.js

```diff
@@ -719,6 +721,18 @@ videojs.appendContent = Dom.appendContent;
  */
 videojs.insertContent = Dom.insertContent;
 
+/**
+ * Export the Dom utilities for use in external plugins
+ * and Tech's
+ */
+videojs.Dom = Dom;
+
+/**
+ * Export the Url utilities for use in external plugins
+ * and Tech's
+ */
+videojs.Url = Url;
```

Comment:
Can we expose this as `url` instead? It's not a constructor; so, the casing doesn't make much sense.